packages/release-please/src/release-please.ts

```diff
@@ -232,6 +237,10 @@ export = (app: Probot) => {
     setLogger(logger);
 
     logger.info(`push (${repoUrl})`);
+
+    // wait 5 seconds for potential GitHub indexing race conditions
+    await delay(DELAY_TIME);
```

Comment:
Is there a way for us to catch this bug in action? I wonder if we could log a metric before the delay? Check our table in bigquery to see if this delay ever stops the behavior observed.